package.json

```diff
@@ -62,10 +62,11 @@
     "@dfinity/candid": "0.15.5",
     "@dfinity/identity": "0.15.5",
     "@dfinity/principal": "0.15.5",
+    "@dfinity/utils": "^0.0.14",
     "bip39": "^3.0.4",
     "lit-html": "^2.3.1",
-    "text-encoding": "^0.7.0",
-    "qr-creator": "^1.0.0"
+    "qr-creator": "^1.0.0",
+    "text-encoding": "^0.7.0"
```

Comment:
what's happened here?

Reply:
`npm i @dfinity/utils` outcome. I guess my npm (v9.6.1) sorted the members alphabetically.